web/mass_class/rtmp_bridge.py

```
@@ -0,0 +1,178 @@
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.processes = {}  # Map of session_id -> subprocess
+
+    async def start_stream(self, event):
+        session_id = event["session_id"]
+        sdp = event["sdp"]
+        teacher_channel = event["teacher_channel"]
+
+        # Create SDP file
+        sdp_path = os.path.join(settings.MEDIA_ROOT, f"sdp_{session_id}.sdp")
+        with open(sdp_path, "w") as f:
```

Ruff (0.8.2)

Async functions should not open files with blocking methods like `open`

(ASYNC230)